src/com/company/possibleOutputOfBall.java

```diff
@@ -0,0 +1,56 @@
+package com.company;
+
+public enum possibleOutputOfBall {
+    Run_0("0 Run"),
+    Run_1("1 Run"),
+    Run_2("2 Run"),
+    Run_3("3 Run"),
+    Run_4("4 Run"),
+    Run_6("6 Run"),
+    Wide("Wide"),
+    NoBall("No Ball"),
+    Wicket("Wicket");
+
+    private final String displayName;
+
+    possibleOutputOfBall(String displayName) {
+        this.displayName = displayName;
+    }
+
+    public String getDisplayName() {
+        return displayName;
+    }
+
+    // Probability of getting Index of possible output for Batsman and Bowler.
+    private static final double[] probabilityArrayOfBatsman = {0.3, 0.25, 0.15, 0.05, 0.12, 0.08, 0.025, 0.015, 0.01};
```

Comment:
probability of wicket and no-ball is not dependent whether the player on strike is batsman or bowler. it should be the same irrespective of who is batting.